Backend/src/modules/tooltip/tests/guild_tooltip.rs

```diff
@@ -0,0 +1,910 @@
+use crate::modules::{
+    armory::{
+        domain_value::{CharacterGear, CharacterGuild, CharacterInfo, GuildRank},
+        material::{Character, CharacterHistory, Guild},
+        tools::GetGuild,
+        Armory,
+    },
+    tooltip::{tools::RetrieveGuildTooltip, Tooltip},
+};
+
+fn build_character_guild(guild_id: u32) -> CharacterGuild {
+    CharacterGuild {
+        guild_id: guild_id,
```

Comment:
I suggest you to get clippy as integration. It helps you to not do non idiomatic patterns. This for example can be simplified by simply `guild_id,`

Reply:
Fixed, I cannot see any other warning in my module that clippy throws.